src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -42,15 +44,21 @@ public void deleted(AbstractBuild r) {
     }
 
     public void started(AbstractBuild build) {
-        String changes = getChanges(build);
-        CauseAction cause = build.getAction(CauseAction.class);
+        CauseAction causeAction = build.getAction(CauseAction.class);
 
+        if (causeAction != null) {
+            Cause scmCause = causeAction.findCause(SCMTrigger.SCMTriggerCause.class);
+            if (scmCause == null) {
+                MessageBuilder message = new MessageBuilder(notifier, build);
+                message.append(causeAction.getShortDescription());
+                notifyStart(build, message.appendOpenLink().toString());
+                return;
```

Comment:
Since this is a `public void` function I will remove the `return` statement as @peergum had mentioned earlier.  I realize that it's not actually returning anything but I think it's a little cleaner.  I'm doing this as post merge to expedite merging your change.